region/lowernorfair/west/Screw Attack Room.json

```diff
@@ -902,6 +906,7 @@
       ],
       "clearsObstacles": ["A", "B", "C"],
       "reusableRoomwideNotable": "Screw Attack Room Descent and Shinespark Escape",
+      "flashSuitChecked": true,
```

Comment:
The ability to use the flash suit to escape might be easier.  You would jump with blue and unmorph to preserve it.  (Entering through middle door with very low speed probably works too).  But that would be written as new strats.